tests/robot-tests/tests/admin_and_public/bau/archive_publication.robot

```diff
@@ -137,8 +140,8 @@ Check that archive-publication subject appears correctly on Data catalogue page
 
     user clicks radio    %{TEST_THEME_NAME}
 
-    user checks page contains    ${PUBLICATION_NAME_ARCHIVE}
-    user checks page does not contain    ${PUBLICATION_NAME_SUPERSEDE}
+    user checks page contains element    //*[@data-testid="Radio item for ${PUBLICATION_NAME_ARCHIVE}"]
+    user checks page does not contain element    //*[@data-testid="Radio item for ${PUBLICATION_NAME_SUPERSEDE}"]
```

Comment:
Could use `testid:` here? If so, can make the same change in a couple of other places in this file.

Reply:
Yeah I think I meant to clean this up first time round but looks like I may have forgotten to replace `//*[@data-testid]` with the `testid:` shorthand syntax

Reply:
I've had a look at implementing this but the tests end up throwing `Page should have contained element 'testid:Radio item for UI tests - UI tests - archived publication 20230113-095242' but did not.`  in certain places. I've switched back to using the `//*[@data-testid=""]` syntax for now as it doesn't really seem worth it to pursue getting the `testId` syntax to work simply for the benefit of syntax sugar.